jsonmerge_utils.py

```diff
@@ -5,17 +5,38 @@
 @author: GRENTOR
 """
 import os
+import argparse
 import logging
 #from tqdm import tqdm
 from merge_files import Merge
+
+parser = argparse.ArgumentParser(description="link-check is a broken link identifier")
+parser.add_argument('-ip',
+                    help="Input prefix",
+                    required=True)

```

Comment:
use the **dest** parameter to make the future reference more inuitive and understandable to others :

```suggestion
parser.add_argument('-ip',
                    dest = "input_prefix"
                    help="Input prefix",
                    required=True)

```

following which the usage would become:
`args.input_prefix`